start.go

```diff
@@ -63,12 +62,12 @@ is a directory with a specification file and a root filesystem.`,
 		}
 
 		if os.Geteuid() != 0 {
-			logrus.Fatal("runc should be run as root")
+			fatalf("runc should be run as root")
```

Comment:
I don't think it needs to get that complicated, should I paid more attention when I reviewed the code :)